app/src/main/res/layout-land/admin_auth_activity.xml

```diff
@@ -32,7 +32,8 @@
         android:background="?attr/colorPrimary"
         android:fontFamily="sans-serif"
         android:minHeight="?attr/actionBarSize"
-        android:textSize="20sp"
+        android:textSize="@dimen/toolbar_title"
```

Comment:
Create dimen value only if it is common and being used at multiple places. 
Recommended to use textsize value directly as 20sp instead of dimen value.

Reply:
ok.